src/components/displayBooks.js

```diff
@@ -1,12 +1,15 @@
 import React from 'react';
+import { useDispatch } from 'react-redux';
 import PropTypes from 'prop-types';
+import { removeBook } from '../redux/books/books';
 
-const DisplayBooks = ({ todos, removeTodo }) => {
+const DisplayBooks = ({ todos }) => {
+  const dispatch = useDispatch();
   const allTodos = todos.map((element) => (
     <li key={element.id} id={element.id}>
-      {element.name}
+      {element.title}
       {' '}
-      <button onClick={removeTodo} type="button">X</button>
+      <button onClick={() => dispatch(removeBook(element.id))} type="button">X</button>
     </li>
```

Comment:
- [ ] OPTIONAL: I recommend you to have a Book component that rendered each book object and accepts the book and onClick props. :pray: I also think that it is better if you name the books array `books` instead of todos here as it is a book store app. :pray: